packages/airnode-node/src/api/index.ts

```diff
@@ -295,9 +297,14 @@ export async function callApi(payload: ApiCallPayload): Promise<LogsData<ApiCall
   const verificationResult = verifyCallApi(payload);
   if (verificationResult) return verificationResult;
 
-  const processedPayload = await preProcessApiSpecifications(payload);
-  const ois = payload.config.ois.find((o: OIS) => o.title === payload.aggregatedApiCall.oisTitle)!;
-  const endpoint = ois.endpoints.find((e: Endpoint) => e.name === payload.aggregatedApiCall.endpointName)!;
+  const {
+    aggregatedApiCall: { parameters },
+  } = payload;
+  const ois = payload.config.ois.find((o) => o.title === payload.aggregatedApiCall.oisTitle)!;
+  const endpoint = ois.endpoints.find((e) => e.name === payload.aggregatedApiCall.endpointName)!;
```

Comment:
Why were the types (and their corresponding imports) removed from this line and the one above it?

Reply:
They were redundant. OIS is typed so there was no need to have them.